scheduler.go

```diff
@@ -199,7 +199,7 @@ func (s *Scheduler) calculateMonths(job *Job, lastRun time.Time) time.Duration {
 				nextRun = nextRun.AddDate(0, int(job.interval), daysDifference)
 			}
 		}
-		return s.until(lastRunRoundedMidnight, nextRun)
+		return s.until(lastRun, nextRun)
```

Comment:
Could you please include a test that fails in the case of  `lastRunRoundedMidnight` and passes in the case of `lastRun`? That would be super helpful.